.github/workflows/maven.yml

```diff
@@ -37,12 +38,15 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-
-      - name: Set up JDK 1.8
+      
+      - name: Set up JDK 1.17
         uses: actions/setup-java@v1
         with:
-          java-version: 1.8
+          java-version: 1.17
```

Comment:
Is this supposed to be `1.17` ? From Java 1.8, a different versioning strategy was used. So it is Java 11, 17, 21 etc.,

Reply:
This is not an issue, still I addressed this. Sharing screenshots for both before & after
![image](https://github.com/user-attachments/assets/001ddad4-b7a3-481a-86c8-d8474242db78)
![image](https://github.com/user-attachments/assets/cc2d1f7e-6e8c-45d2-bb12-6a2541688790)